libvips/foreign/libnsgif/Makefile.am

```diff
@@ -12,7 +12,6 @@ endif
 
 EXTRA_DIST = \
 	meson.build \
-	patches \
```

Comment:
`patches` is needed for `Makefile.am`. I think it's harmless to have it here and empty.

Reply:
Sure, sounds like a good idea.

Reply:
OK, done: https://github.com/libvips/libvips/pull/2889/commits/e4c5039f61ed7f3c98462f5d2a86d5376c4a1ed6.